mock-core-configs/etc/mock/templates/epel-9.tpl

```diff
@@ -50,8 +50,8 @@ gpgcheck=1
 gpgkey=file:///usr/share/distribution-gpg-keys/epel/RPM-GPG-KEY-EPEL-$releasever
 skip_if_unavailable=False
 
-[local]
-name=local
+[local-epel]
+name=local-epel
```

Comment:
Would you mind keeping local here?  It would minimize changes...

IMO we can change only the `local-epel-next` below (or even say `local-next`).

Reply:
It conflicts with `local` from CS9 base, so no, we can't.

Reply:
Ah, right.